addons/web/static/src/views/form/form_group/form_group.xml

```diff
@@ -2,44 +2,42 @@
 <templates xml:space="preserve">
 
 <t t-name="web.Form.OuterGroup" owl="1">
-  <div class="o_group" t-attf-class="{{ allClasses }}" t-att-style="props.style">
-    <t t-slot="title" t-if="props.slots.title" />
-    <t t-foreach="getItems()" t-as="item" t-key="item_index" >
-      <br t-if="item.newline" />
-      <t t-slot="{{ item.name }}" className="'o_group_col_' + item.size"/>
-    </t>
-  </div>
+    <div class="o_group row g-2 gx-lg-5 align-items-start" t-attf-class="{{ allClasses }} {{ props.hasOuterTemplate and props.class ? '' : 'my-3' }}" t-att-style="props.style">
+        <t t-slot="title" t-if="props.slots.title" />
+        <t t-foreach="getItems()" t-as="item" t-key="item_index">
+            <span class="o_newline" t-if="item.newline" />
+            <t t-slot="{{ item.name }}" className="item.colspan !== props.maxCols ? 'col-lg-' + item.size : ''"/>
+        </t>
+    </div>
 </t>
 
 <t t-name="web.Form.InnerGroup" owl="1">
-  <table t-attf-class="{{ allClasses }}" class="o_inner_group o_group" t-if="props.slots" t-att-style="props.style">
-      <tbody>
-          <td t-if="props.slots.title" t-attf-colspan="{{ props.maxCols }}" style="width: 100%">
+    <div t-attf-class="{{ allClasses }} {{ props.class and props.class.indexOf('oe_subtotal_footer') >=0 ? 'd-grid gap-0 text-end' : 'd-sm-grid' }}" class="o_inner_group" t-if="props.slots" t-att-style="props.style">
+        <div t-if="props.slots.title" t-attf-style="grid-column: span {{ props.maxCols }}">
               <t t-slot="title" />
-          </td>
-          <tr t-foreach="getRows()" t-as="row" t-key="row_index">
-            <t t-foreach="row" t-as="cell" t-key="cell_index" >
+        </div>
+        <div t-foreach="getRows()" t-as="row" t-key="row_index" class="o_wrap_field mb-3 mb-sm-0" t-attf-class="{{ props.class and props.class.indexOf('oe_subtotal_footer') >=0 ? 'd-contents' : 'd-flex d-sm-contents flex-column' }}" t-if="row.isVisible">
```

Comment:
Not sure if it's the line causing the issue, but on mobile the (visual) distance between fields is lower that the distance between labels and inputs.. it makes very difficult to scan.
It was the case already before, but here seems a bit worst 

<img width="192" alt="image" src="https://user-images.githubusercontent.com/8558351/185955355-9c8c0c80-45c3-4b44-affe-462f2e09e8ba.png">

Reply:
Fixed. I put the style of label on mobile back and adapted the style of `o_horizontal_separator` to increase the readability.